src/client.ts

```diff
@@ -173,6 +209,10 @@ export class ScrapflyClient {
       }
       throw new errors.ApiHttpClientError(JSON.stringify(data));
     }
+
+    const content = await this.handleLargeObjects(data.result)
```

Comment:
Same as for the python SDK, https://github.com/scrapfly/python-scrapfly/pull/23#discussion_r1713845937